website/addons/base/__init__.py

```diff
@@ -772,14 +772,42 @@ def after_set_privacy(self, node, permissions):
         pass
 
     def before_fork(self, node, user):
+        """Return warning text to display if user auth will be copied to a
+        fork.
+        :param Node node:
+        :param Uder user
+        :returns Alert message
         """
 
-        :param Node node:
-        :param User user:
-        :returns: Alert message
+        if hasattr(self, "user_settings"):
+            if self.user_settings is None:
+                return (
+                    u'Because you have not configured the authorization for this {addon} add-on this '
+                    u'{category} will not transfer your authentication token to '
```

Comment:
"authentication token" is too specific--not all addons may use token-based auth. How about: "will not transfer you authentication to the forked {category}".